src/multisignature.cpp

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) 2017-2018 The PIVX developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <regex>
+#include <boost/algorithm/string/split.hpp>
+#include "multisignature.h"
+#include "utilstrencodings.h"
+#include "tinyformat.h"
+#include "util.h"
+#include "streams.h"
+#include "wallet.h"
+
+
+using namespace std;
```

Comment:
Please don't introduce new source files with `using namespace ...` for `std` like this. instead, explicitely use the namespace in-line:
```cpp
std::string strRedeemScript
```
instead of
```cpp
using namespace std;

string strRedeemScript
```